### PR TITLE
Drop multipool autoswap

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
refs: #3860

## Description

Remove multipoolAutoswap

### Security Considerations

none

### Documentation Considerations

Documentation still needs to be removed.

### Testing Considerations

Tests have been removed or updated.

### CODE REVIEW

File: packages/zoe/src/contracts/vpool-xyk-amm/priceAuthority.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
 import { observeNotifier } from '@agoric/notifier';
 
-import { makeOnewayPriceAuthorityKit } from '../../contractSupport/index.js';
+import { makeOnewayPriceAuthorityKit } from '../../contractSupport';
```
Comment (dckc) [inline packages/zoe/src/contracts/vpool-xyk-amm/priceAuthority.js]: Was this change on purpose? I think bundleSource won't like it.

**Review by dckc — CHANGES_REQUESTED** [review]
I don't see how the priceAuthority.js change can work without `/index.js`

Otherwise, yay! 6500 LOC goes bye bye!

File: packages/zoe/src/contracts/vpool-xyk-amm/priceAuthority.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/zoe/src/contracts/vpool-xyk-amm/priceAuthority.js]: Not on purpose. Must have been done automatically when moving the file.  I think prettier is always suggesting shortening names like this, and I never manually accept, but I'll bet it can automatically fix it when I'm not looking in some cases.

**Review by dckc — APPROVED** [review]
